### DISCUSSION TITLE
v4.1.0 release

### DISCUSSION THREAD

**Discussion by andyzhangx** [kubernetes-csi/csi-driver-nfs#357] [Announcements]
### Features 🌈
 - skip chmod if mountPermissions is 0 after mount (#328)
 - support IPv6 server address (#342)
 - add affinity, nodeSelector, runOnControlPlane in chart config (#347)
 - add subDir in storage class parameters (#349)
 - support pv/pvc metadata in subDir parameter (#353)

### Bug Fixes 🐞
 - fix: add serviceAccount for driver daemonset running on the node (#335)
 - fix: bypass chmod if mounting point permissions are correct (#340)
 - fix: incorrect runOnMaster config (#345)

## What's Changed
* test: run k8s 1.24 external e2e test by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/333
* fix: add serviceAccount for driver daemonset running on the node by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/335
* chore: remove hard mount option by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/336
* fix: bypass chmod if mounting point permissions are correct by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/340
* chore: refine bypass chmod code by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/341
* feat: support IPv6 server address by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/342
* chore: update node-driver-registrar,liveness to latest version by @humblec in https://github.com/kubernetes-csi/csi-driver-nfs/pull/343
* fix: incorrect runOnMaster config by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/345
* master: update release-tools by @pohly in https://github.com/kubernetes-csi/csi-driver-nfs/pull/346
* feat: add affinity, nodeSelector, runOnControlPlane in chart config by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/347
* feat: add subDir in storage class parameters by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/349
* fix: add missing runOnControlPlane config in chart by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/350
* feat: support pv/pvc metadata in subDir parameter by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/353
* chore: upgrade sidecar image versions by @andyzhangx in https://github.com/kubernetes-csi/csi-driver-nfs/pull/355

**Full Changelog**: https://github.com/kubernetes-csi/csi-driver-nfs/compare/v4.0.0...v4.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kubernetes-csi/csi-driver-nfs/releases/tag/v4.1.0'>v4.1.0 release</a>.</em>

**Comment by luisrosilva** [comment]
Should images already be in the registry? Apparently `https://k8s.gcr.io/v2/sig-storage/nfsplugin/manifests/v4.0.0` exists but `https://k8s.gcr.io/v2/sig-storage/nfsplugin/manifests/v4.1.0` doesn't.

**Reply by andyzhangx** [reply]
it depends on https://github.com/kubernetes/k8s.io/pull/3923
